### ISSUE TITLE
YOLOv5 Tutorial Assistance

### ISSUE THREAD

**Issue by glenn-jocher** [Deci-AI/super-gradients#301]
@shaydeci @oferbaratz @ofrimasad hi I'm working on our new Deci.ai + YOLOv5 partnership tutorial and need some help. The tutorial is at https://github.com/ultralytics/yolov5/wiki/YOLOv5-Deci-AI-Tutorial and is based on a word document provided by Rachel than I've converted to Markdown. 

I'd like you guys at Deci to review the content and help supply the 6 additional images (denoted by **IMAGE_HERE**). To streamline this I've pasted the markdown content directly here in this issue. If you simply edit this issue with the appropriate changes I can then transfer those over to the YOLOv5 repo.

For the images I've provided one example hyperlinked image myself. The guidelines are that they should be 1920 pixel wide JPG screenshots at <500kB each.

Thanks for the help and let me know if you have any questions!

# Markdown content below

📚 This guide explains how to streamline the process of **compiling and quantizing** YOLOv5 🚀 to achieve better performance with the **[Deci platform](https://deci.ai/platform/?utm_source=Github&amp;utm_medium=Ultralytics&amp;utm_campaign=deci-signup-yolov5&amp;utm_content=platform)**. UPDATED 6 August 2022.

## Content

- About the Deci Platform
- First-time setup
- Runtime optimization and benchmarking of your model

## About Deci Platform

The Deci platform includes free tools for easily managing, optimizing, and deploying models in any production environment. Deci supports all popular DL frameworks, such as TensorFlow, PyTorch, Keras and ONNX. All you need is our web-based platform or our Python client to run it from your code.

#### With Deci you can:

- **Improve Inference performance by up to 10X**  
Automatically compile and quantize your models and evaluate different production settings to achieve better latency, throughout, reduce model size and memory footprint on your hardware.

- **Find the best inference hardware for your application**  
Benchmark your model's performance on various hardware (including edge) devices with a click of a button. Eliminate the need to manually setup and test various hardware and production settings.

- **Deploy with a Few Lines of Code**  
Leverage Deci's python-based [inference engine](https://docs.deci.ai/docs/infery?utm_source=Github&amp;utm_medium=Ultralytics&amp;utm_campaign=deci-signup-yolov5&amp;utm_content=infery). Compatible with multiple frameworks and hardware types.

For more information about the Deci platform please visit Deci's [website](https://deci.ai/platform/?utm_source=Github&amp;utm_medium=Ultralytics&amp;utm_campaign=deci-signup-yolov5&amp;utm_content=platform).

### Step 1: 

Go to [https://console.deci.ai/sign-up](https://console.deci.ai/sign-up?utm_source=Github&amp;utm_medium=Ultralytics&amp;utm_campaign=deci-signup-yolov5&amp;utm_content=sign-up) and open your free account. 

<a href="https://console.deci.ai/sign-up?utm_source=Github&amp;utm_medium=Ultralytics&amp;utm_campaign=deci-signup-yolov5&amp;utm_content=sign-up">
<img alt="Deci AI signup page" src="https://user-images.githubusercontent.com/26833433/183249047-68577230-44f5-422c-8fe1-b000d2b11d4b.jpg" width="1280"/></a>

### Step 2:

In order to start optimizing your pre-trained YOLOv5 model, you will need to convert it into an ONNX format. Please follow these simple instructions on [this link](https://github.com/ultralytics/yolov5/issues/251) to convert your model to ONNX format.

### Step 3:

Go to "Lab" tab and click the "New Model" button in the top right part of the screen to upload your model in the ONNX format.

<img alt="Deci AI Lab page" src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/assets/SG_img/Deci_img/Deci_lab.png" width="1280"/>

Follow the steps of the model upload wizard to select your target hardware as well as desired batch size and quantization level for the model compilation.

<img alt="Deci AI Lab page" src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/assets/SG_img/Deci_img/Deci_add_model.png" width="1280"/>

After filling in the relevant information, click "Start". The Deci platform will automatically perform a runtime optimization of your YOLOv5 model for the hardware you selected as well as benchmark your model on various hardware types. This process takes approximately 10 minutes.

Once done, a new row will appear on your screen underneath the baseline model you previously uploaded. Here you can see the optimized version of your pre-trained YOLOv5 model.

<img alt="Deci AI Lab page" src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/assets/SG_img/Deci_img/Deci_runtime_optimization.png" width="1280"/>

## What's next?

1. You can then download your optimized model by clicking on "Deploy" button

<img alt="Deci AI Lab page" src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/assets/SG_img/Deci_img/Deci_deploy_button.png" width="1280" />

You will then be prompted to download your model and receive the instructions on how to install and use Infery - Deci's runtime inference engine.

The use of Infery is optional. You can get the python raw files and use them with any other inference engine of your choice.

<img alt="Deci AI Lab page" src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/assets/SG_img/Deci_img/Deci_deploy_model.png" width="1280"/>

2. Explore the optimization and benchmark results on the "Insights" tab.

<img alt="Deci AI Lab page" src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/assets/SG_img/Deci_img/Deci_insights.png" width="1280"/>

**Comment by oferbaratz** [comment]
Hey @glenn-jocher
I've attached the relevant images and edited your markdown. Please let me know if you need any other assistance

**Comment by glenn-jocher** [comment]
@oferbaratz thanks! I changed the /blob/ URLs to /raw/ for the images to render correctly, looks good.
